### PR TITLE
docs: clean up OIDC duplicates

### PR DESCRIPTION
**Pull request by ericpaulsen**
our OIDC docs had steps for both Google and GitLab, which were effectively the same config. I reduced it to just one section and kept it generic, as the env vars can be used for any OIDC provider. also added example Helm configs.

### CODE REVIEW

**Comment by Emyrk** [comment]
I had wondered about this when I setup OIDC myself.

File: docs/admin/auth.md [hunk 1]
```diff
@@ -50,88 +50,88 @@ CODER_OAUTH2_GITHUB_ALLOW_EVERYONE=true
...
-### Step 2: Configure Coder with the Google OpenID Connect credentials
-
-Navigate to your Coder host and run the following command to start up the Coder
-server:
+To upgrade Coder, run:
 
 ```console
-coder server --oidc-issuer-url="https://accounts.google.com" --oidc-email-domain="your-domain-1,your-domain-2" --oidc-client-id="533...ent.com" --oidc-client-secret="G0CSP...7qSM"
+helm upgrade <release-name> coder-v2/coder -n <namespace> -f values.yaml
 ```
```
Comment (Emyrk) [inline docs/admin/auth.md:133]: We should still show how to configure with non-helm. No?

Comment (ericpaulsen) [reply docs/admin/auth.md:133]: we do, in line 97, unless i'm missing something? @Emyrk

**Review by spatecon — APPROVED** [review]

File: docs/admin/auth.md (hunk 1 above)
Comment (Emyrk) [reply docs/admin/auth.md:133]: I am blind lol

**Review by bpmct — APPROVED** [review]

**Pull request merged**